src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -305,6 +305,9 @@
   - name: --attach-acr
     type: string
     short-summary: Grant the 'acrpull' role assignment to the ACR specified by name or resource ID.
+  - name: --api-server-authorized-ip-ranges
+    type: str
+    short-summary: Comma seperated list of authorized apiserver IP ranges. Set to "" to disable. If left unset, all external traffic will be allowed.
```

Comment:
"Set to "" to disable. If left unset, all external traffic will be allowed." feels like 2 options, but they are the same thing, right?

Reply:
The empty string is to disable IPWL for a previously enabled cluster. Since IPWL isn't a default behavior we should only let users set "" for IPWL on update

Reply:
Removed as requested